File: Excel_Sheet_Column_Title.cpp
class Solution {
public:
    string convertToTitle(int columnNumber) {
        string res = "";

        while (columnNumber > 0) {
            columnNumber--;
            res = char((columnNumber % 26) + 'A') + res;
            columnNumber /= 26;
        }
        
        return res;        
    }
};

File: Excel_Sheet_Column_Title.js
/**
 * @param {number} columnNumber
 * @return {string}
 */
var convertToTitle = function(columnNumber) {
    let res = "";

    while (columnNumber > 0) {
        columnNumber--;
        res = String.fromCharCode((columnNumber % 26) + "A".charCodeAt(0)) + res;
        columnNumber = Math.floor(columnNumber / 26);
    }
    
    return res;    
};
